text/0111-feature-flags.md

```diff
@@ -154,9 +154,30 @@ describe('foo', () => {
 });
 ```
 
+Note that this will only work for features behind flags that are not used during the initialization of the engine.
```

Comment:
are there any examples of initialization features that we can list?

Reply:
We don't have anything explicit like what you're asking for, but any code that is initially evaluated when the engine loads are subject to this restriction. This includes variable initializations and function declarations.